src/rhsmlib/dbus/server.py

```diff
@@ -31,16 +31,12 @@
 from rhsmlib.file_monitor import CONSUMER_WATCHER, ENTITLEMENT_WATCHER, CONFIG_WATCHER, PRODUCT_WATCHER, \
     SYSPURPOSE_WATCHER
 from subscription_manager import injection as inj
-from rhsm.logutil import init_logger
-
 
 log = logging.getLogger(__name__)
 
 parser = get_config_parser()
 conf = config.Config(parser)
 
-init_logger(parser)
```

Comment:
Calling all of this here is wrong, but simple removing this code had bad side effect. We lost logging of rhsm.service. I would move it to some method (maybe `__init__()` of `Server` class).

Reply:
moved to the __init__() of Server class.